apps/dynatrace/dynatrace-crds-upgrade-1.0.1/kustomize.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
+kind: Kustomization
+metadata:
+  name: dynatrace-crd
+  namespace: flux-system
```

Comment:
is namespace correct?

Reply:
yes the namespace is correct, as this is the namespace that exists in the Kubernetes cluster